src/data/automation.ts

```diff
@@ -113,6 +113,7 @@ export interface EventTrigger {
   platform: "event";
   event_type: string;
   event_data: any;
+  context: any;
```

Comment:
This is not true. Right now context is always `context_id: string, user_id?: string`